### PR TITLE
flutter: add channel and run all example

### PR DESCRIPTION
**Pull request by martinloesethjensen**
Page edit

- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples. 
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/flutter.md [hunk 1]
```diff
@@ -22,3 +22,11 @@
 - Show information about the installed tooling:
 
 `flutter doctor`
+
+- List or change Flutter channel. Channels: stable, beta, dev, master:
+
+`flutter channel {{channel}}`
```
Comment (sbrl) [inline pages/common/flutter.md:28]: If there are only a select set of options available, we can use this instead:
```suggestion
- List or change Flutter channel:

`flutter channel {{stable|beta|dev|master}}`
```

**Review by einverne — COMMENTED** [review]
Looks good after @sbrl 's suggestion.

**Comment by martinloesethjensen** [comment]
@sbrl @einverne Thanks for the suggestion :)

**Review by einverne — APPROVED** [review]
Looks good to me. Thanks @martinloesethjensen .

**Review by sbrl — APPROVED** [review]
Thanks, @martinloesethjensen :heart:

**Pull request merged**